src/components/HackMercedApplication/index.tsx

```diff
@@ -199,8 +203,8 @@ const Application: FC = (): JSX.Element => {
                             <label htmlFor="gender">
                                 Gender <span className="required">*</span>
                             </label>
-                            <select id="gender" name="gender" onChange={handleInputChange} required>
-                                <option value="" disabled={true} selected={true}>
+                            <select id="gender" name="gender" onChange={handleInputChange} required defaultValue="">
+                                <option value="" disabled={true}>
                                     Gender
                                 </option>
                                 <option value="male">Male</option>
```

Comment:
Use non-binary over other, it's more inclusive and less othering (literally)

Reply:
ok